Sources/AblyAssetTrackingPublisher/PublisherTrackableState.swift

```diff
@@ -0,0 +1,44 @@
+ //
+//  Created by Łukasz Szyszkowski on 25/08/2021.
+//
+
+import Foundation
+
+struct PublisherTrackableState {
+    
+    private enum Constants {
```

Comment:
Why is this an enum? And im not sure, but would it make sense to make it customizable?

Reply:
- enum because you can't initialize it (you don't have to create `private init`)
- this is an internal class so `maxRetryCount` could be helpful for the future use and additionally I'm using it in tests

Reply:
I meant why not just create a `private let maxRetryCount  = 1` inside `PublisherTrackableState`

Reply:
On second thought, I agree with you.
I used `Constants` "namespace" to separate this from other variables in this class, this is how we do it in my company, but, as we can see, this is unnecessary. I'll change it.

Reply:
3f490050915edcc38c1acb1410e3c68562b80e60